Clappr.xcodeproj/xcshareddata/xcschemes/Clappr.xcscheme

```diff
@@ -16,7 +16,7 @@
                BuildableIdentifier = "primary"
                BlueprintIdentifier = "96D3627B1D41337600CCB866"
                BuildableName = "Clappr.framework"
-               BlueprintName = "Clappr"
+               BlueprintName = "Clappr-iOS"
```

Comment:
@igoriols it seems this scheme wasn't really renamed, can you double check this?